hw03_babayemie.py

```diff
@@ -0,0 +1,132 @@
+#################################################################################
+# Author: Elijah Babayemi
+# Username: babayemie
+#
+# Assignment:HW03: Fully Functional Gitty Psychedelic Robotic Turtles Purpose: Continue practicing creating and using
+# functions, more practice on using the turtle library, learn about how computers represent colors, and learn about
+# source control and Git.
+#
+# Google Doc Link: https://docs.google.com/document/d/1X9d90s3RxZa3LG8GFm7Y8XtmVSOv1ai_OpNtu0xGtpk/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+# https://openbookproject.net/thinkcs/python/english3e/hello_little_turtles.html
+# https://www.geeksforgeeks.org/draw-color-filled-shapes-in-turtle-python/
+#################################################################################
+
+
+import turtle
+
+
+def make_house(house_turtle):
+    """
+  This function draws the main body of the house
+   """
+    house_turtle.penup()
+    house_turtle.goto(-50, -50)
+    house_turtle.pendown()
+
+    house_turtle.fillcolor("#FF3C00")  # red color
+    house_turtle.begin_fill()
+    for _ in range(4):
+        house_turtle.forward(100)
+        house_turtle.left(90)
+    house_turtle.end_fill()
+
+    # Draws a door for house
+    house_turtle.penup()
+    house_turtle.fillcolor("#060D04")
+    house_turtle.begin_fill()
+    house_turtle.goto(-15, -50)
+    for _ in range(4):
+        house_turtle.forward(30)
+        house_turtle.left(90)
+    house_turtle.end_fill()
+    house_turtle.hideturtle()
+
+
+def draw_roof(shape):
+    """
+   This function draws a roof for the square to make it look like a house.
+   """
+    shape.penup()
+    shape.fillcolor("#898685")
+    shape.begin_fill()
+    shape.goto(50, 50)
+    shape.pendown()
+    shape.goto(0, 100)
+    shape.goto(-50, 50)
+    shape.goto(50, 50)
+    shape.end_fill()
+    shape.hideturtle()
+
+
+def draw_tree(tree_turtle):
+    """
+   This function draws a tree beside the house.
+   """
```

Comment:
These two lines are indented one space less than the rest. Fix them (they didn't cause a problem here, but single-space indentation have ruined weekends for me in the past).